docs/docs/how-to-deploy.mdx

```diff
@@ -136,6 +138,87 @@ The `<custom_image_tag>` should reference how this image will be different from
 example, you could version or date your tags, as well as create different tags that have different code for production
 and development servers. You should create a new tag any time you update your code and want to re-deploy it.
 
+#### Automating your Action Server Image Builds
+
+In addition to a manual step to create a new Action Server Image. You can use [Rasa Action Server GitHub Action](https://github.com/RasaHQ/action-server-gha) to automate image builds.
+If GitHub Actions are new to you it might be helpful to get familiar with [GitHub Actions Documentation](https://docs.github.com/en/actions).
+
+The following steps assume that you already created a GitHub repository and you have a DockerHub account.
+
+To create a workflow for building and pushing a Docker image into a DockerHub registry:
+
+1. Create GitHub Secrets with login name and password for DockerHub.
+   [Here](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) you can find details on how to create encrypted secrets for a repository.
+
+   The example uses the following secrets:
+   - `DOCKER_HUB_LOGIN` - a login name for DockerHub
+   - `DOCKER_HUB_PASSWORD` - a password for DockerHub
+
+2. In your GitHub repository create a file [`.github/workflows/action_server.yml`](https://github.com/RasaHQ/action-server-gha/blob/master/examples/action_server.yml).
```

Comment:
The GitHub link 404s

Reply:
Yes, that's ok. After the https://github.com/RasaHQ/action-server-gha/pull/1 is merged, the link should work.